lib/utils.js

```diff
@@ -936,4 +936,6 @@ exports.errorToPOJO = function errorToPOJO(error) {
   return ret;
 };
 
-exports.nodeMajorVersion = parseInt(process.versions.node.split('.')[0], 10);
+exports.nodeMajorVersion = function nodeMajorVersion() {
```

Comment:
Maybe we make `nodeMajorVersion()` return `null` if `process.versions` is undefined?

Reply:
Sure! Can do

Reply:
Waiting with this change until the convo below is resolved. :arrow_down: